introduction.tex

```diff
@@ -5,6 +5,8 @@ \chapter{Introduction}
 TODO: Make another pass on that section after everything else is redacted (or
 at least the soa)\\
 
+something silly
+
 The need for scalable computing infrastructure has increased tremendously in
 the last decades. Nearly every field of computer science, from research to the
```

Comment:
If you can quantify is better, has increased from  peta to exascale...

Reply:
Even public sector... This manifest your opinion that public sector is somewhat less updated than others, I would drop the even